Abstract (#23)
greenelab/manubot-gpt-manuscript: content/01.abstract.md

## Abstract {.page_break_before}

Academics often communicate through scholarly manuscripts, which describe new advances, summarize existing literature, or argue for changes in the status quo.
However, writing and revising manuscripts is a time-consuming process.
Large language models are bringing new capabilities to many areas of knowledge work that have shown impressive performance in different tasks.

> We integrated the use of these models into the Manubot publishing ecosystem to automatically revise and improve existing text.

Reviewer: ```suggestion
We integrated these models into the Manubot publishing ecosystem to suggest revisions for scholarly text.
```

The user can run a workflow that will trigger a series of queries to OpenAI's language models to create a timestamped set of suggested revisions to the text.

> We tested this AI-based revision workflow in existing manuscripts, including the present one.

Reviewer: ```suggestion
We tested this AI-based revision workflow in three case studies of existing manuscripts, including the present one.
```

> We found that these models can understand the concepts in scholarly text and produce high-quality revisions that significantly improved the reading.

Reviewer: ```suggestion
We found that these models can capture the concepts in the scholarly text and produce high-quality revisions that improved clarity
```
